src/conn/conn_layered.c

```diff
@@ -892,6 +892,13 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
               session, ret = __wt_disagg_begin_checkpoint(session, next_checkpoint_id));
             WT_ERR(ret);
         }
+
+        WT_ERR(__wt_config_gets(session, cfg, "disaggregated.internal_page_delta", &cval));
```

Comment:
perhaps this can go before the check
```

    if (reconfig)
        return (0);
```
This is reconfigurable.

Reply:
Changed this [here](https://github.com/wiredtiger/wiredtiger/pull/11743/commits/d3d9ca4c168974c73eacb1f84a6c07a40053eafb).

Reply:
I'm ambivalent about making this runtime configurable at the moment - I believe it would work fine to switch it at runtime at the moment, but I'd also prefer to avoid the overhead of needing to test and ensure switching at runtime works.

Reply:
I'm not quite following what you mean by this comment, would you mind elaborating? @agorrod

Reply:
I'd be happy if the new delta control options didn't support changes via WT_CONNECTION::reconfigure at the moment. I understand that it will work given the current implementation, so I don't mind if runtime changes are supported.